### PR TITLE
Fix the pinned toolchain version handling for the ARM64 build.

### PR DESCRIPTION
**Pull request by hjyamauchi**

### CODE REVIEW

**Comment by hjyamauchi** [comment]
@swift-ci please test

File: utils/build.ps1 [hunk 1]
```diff
@@ -774,7 +774,7 @@ function Get-PinnedToolchainVersion() {
   if (Test-Path variable:PinnedVersion) {
     return $PinnedVersion
   }
-  return "5.10.1"
+  throw "PinnedVersion must be set"
```
Comment (compnerd) [inline utils/build.ps1:777]: I don't get the body of this function - the `variable:PinnedVersion` expansion is being tested as a path, but not relative to any location, so how does this work?

Comment (hjyamauchi) [reply utils/build.ps1:777]: I think `Test-Path variable:FOO` is an idiom to check if variable `FOO` is set.

**Comment by hjyamauchi** [comment]
@swift-ci please test

**Review by compnerd — APPROVED** [review]

**Comment by hjyamauchi** [comment]
@swift-ci please test

**Comment by hjyamauchi** [comment]
@swift-ci please test Linux platform

**Comment by hjyamauchi** [comment]
@swift-ci please test Windows platform

**Pull request merged**